backend/src/estark/json_exporter/mod.rs

```diff
@@ -197,7 +197,7 @@ fn fixup_name(name: &str) -> String {
     if name.contains('.') {
```

Comment:
This cannot be the case any more, but I think the part you changed should remain as it is, shouldn't it? It replaces the last `::` by a `.`.

Reply:
It was not being replaced correctly. With the original code, `a::b::c` becomes `a::b.:c`, with this change, it now returns `a::b.c` which I understand is the correct version.